pkgs/development/interpreters/lua-5/build-lua-package.nix

```diff
@@ -21,7 +21,7 @@
 
 # Dependencies needed for running the checkPhase.
 # These are added to buildInputs when doCheck = true.
```

Comment:
This should probably refer to `nativeBuildInputs` now.

Reply:
yes, thanks